Wikipedia/Code/ExploreViewController.swift

```diff
@@ -532,6 +527,37 @@ extension ExploreViewController: ReadingListsAlertControllerDelegate {
     }
 }
 
+private extension WMFContentGroupKind {
+    var hideCardsActionTitle: String {
+        switch self {
+        case .featuredArticle:
+            return WMFLocalizedString("explore-feed-preferences-hide-featured-article-action-title", value: "Hide all Featured article cards", comment: "Title for action that allows users to hide all Featured article cards")
```

Comment:
These strings could be a format - "Hide all %@ cards" with the existing titles (or new ones)